docs/msgextras.md

```diff
@@ -0,0 +1,66 @@
+---
+id: msgextras
+title: Message Extras
+---
+
+Extras in messages can be used to extend clients. This page lists properties supported by the official gotify/\* clients.
+
+Extras are stored in a key-value scheme and are only accepted in `POST /message` requests
+with `application/json` content-type. Keys should be in the following format: `<top-namespace>::[<sub-namespace>::]<action>`
+
+Here's an example how the message with extras looks like. More information can be found in the [swagger-docs](../api-docs).
```

Comment:
```suggestion
Here's an example of how the message with extras looks like. For example, this message could be received by a home automation server, which decodes the extras fields as defined and perform automated tasks as defined by the user.
```

I think the swagger docs did not say anything more than this documentation. No need to refer it to the user again.

I also added a short description of how the extras of this example might works.